pywbem/_cim_operations.py

```diff
@@ -1599,6 +1602,39 @@ def _activate_logger(cls, logger_name, simple_name, detail_level, handler,
                 # TODO execute stage of wbem connection if not already executed.
                 # How do we know it is already executed???
 
+    def _verify_open(self):
+        """
+        Verify that this connection is open and raise ConnectionError otherwise.
+
+        Raises:
+          :exc:`~pywbem.ConnectionError`: Connection must be open but is closed
+        """
+        if self.session is None:
+            raise ConnectionError(
+                "Connection must be open but is actually closed",
+                conn_id=self.conn_id)
+
+    def close(self):
+        """
+        Closes this connection.
+
+        *New in pywbem 1.2.*
+
+        The connection must be open. This method closes the connection and
```

Comment:
Propose slight reordering here:   This method closes the connection in the underlying requests package causing it to close any sockets it has pooled and marks the connection closed. Any subsequent WBEM connection operation requests will generate an exception.

Reply:
DONE - Changed the paragraph to:
"This method closes the session in the underlying requests package,
causing it to close any sockets it has pooled and marks the connection
closed. Any subsequent WBEM connection operation requests will generate
an exception."